schemas/registration.json

```diff
@@ -0,0 +1,123 @@
+{
+  "name": "registration",
+  "description": "Collection of elements required to register the required Donor-Specimen-Sample data to the ARGO Platform. Registration of samples is required before genomic and clinical data submission can proceed.",
+  "key": "submitter_donor_id",
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "restrictions": {
+        "required": true      }
+    },
+    {
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "gender",
+      "valueType": "string",
+      "description": "Description of the donor self-reported gender. Gender is described as the assemblage of properties that distinguish people on the basis of their societal roles.",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Male",
+          "Female",
+          "Other"
+        ]
+      }
+    },
+    {
+      "name": "specimen_submitter_id",
+      "valueType": "string",
+      "description": "Unique identifier of the specimen, assigned by the data provider.",
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "specimen_type",
+      "valueType": "string",
+      "description": "Tissue source of the biospecimen.",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Blood derived",
+          "Blood derived - bone marrow",
+          "Blood derived - peripheral blood",
+          "Bone marrow",
+          "Buccal cell",
+          "Lymph node",
+          "Solid tissue",
+          "Plasma",
+          "Serum",
+          "Urine",
+          "Cerebrospinal fluid",
+          "Sputum",
+          "NOS (Not otherwise specified)",
+          "Other",
+          "FFPE",
+          "Pleural effusion",
+          "Mononuclear cells from bone marrow",
+          "Saliva",
+          "Skin"
+        ]
+      }
+    },
+    {
+      "name": "tumor_normal_designation",
+      "valueType": "string",
+      "description": "Description of the kind of tissue that was collected with respect to disease status.",
+      "restrictions": {
+        "required": true,
+        "codeList": [
```

Comment:
suggest to use exact codeList used by ICGC dictionary: https://docs.icgc.org/dictionary/viewer/#?viewMode=details&dataType=specimen. There were a lot of thoughts went into the design of the codeList, terms are intuitive and formative, also all terms are mutually exclusive.

Reply:
I used the list from @hknahal clincal excel - Hardeep, do you see an issue with expanding this to include the other ICGC terms?

Reply:
During discussions with Peter and the working group, there was disagreement on using the ICGC "specimen_type" codelist because it mixes two concepts - tissue source and tumour/normal designation. For this reason, we decided to separate the two concepts into two separate fields.

Reply:
OK, I see. Now with `specimen_type` changed to `specimen_tissue_source`, terms became less confusing.

It's still necessary to change `tumor_normal_designation` to `tumour_normal_designation`, just to be consistent (all GB spelling).

Reply:
my american is showing :p i thought i had got them all.  this is corrected.